travis/install_dependencies.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+if [[ $VERSION_ES == "2.x" ]]; then
+    sudo apt-get autoremove --purge elasticsearch
+    wget -qO - https://packages.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+    echo "deb http://packages.elastic.co/elasticsearch/2.x/debian stable main" | sudo tee -a /etc/apt/sources.list.d/elk.list
+    sudo apt-get update && sudo apt-get install elasticsearch -y
+    sudo service elasticsearch start
+    sed -i "s/\('elasticsearch\)\(.*\)\('\)/\1 >2\3/g" setup.py
```

Comment:
On 04/27/2016 08:27 PM, BoySka wrote:
> In travis/install_dependencies.sh
> <https://github.com/insomnia-lab/libreant/pull/281#discussion_r61311207>:
> 
>> @@ -0,0 +1,11 @@
>> +#! /bin/bash
>> +
>> +if [[ $VERSION_ES == "2.x" ]]; then
>> +    sudo apt-get autoremove --purge elasticsearch
>> +    wget -qO - https://packages.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
>> +    echo "deb http://packages.elastic.co/elasticsearch/2.x/debian stable main" | sudo tee -a /etc/apt/sources.list.d/elk.list
>> +    sudo apt-get update && sudo apt-get install elasticsearch -y
>> +    sudo service elasticsearch start
>> +    sed -i "s/\('elasticsearch\)\(.*\)\('\)/\1 >2\3/g" setup.py
> 
> what a |sed| expression! what does it do?
> 

It simply change the ealsticsearch dependency version in `setup.py` from
`elasticsearch >=1, <2` to `elasticsearch >2`.